packages/components-native/src/ProgressBar/ProgressBar.size.style.ts

```diff
@@ -0,0 +1,8 @@
+import { StyleSheet } from "react-native";
+import { tokens } from "../utils/design";
+
+export const sizeStyles = StyleSheet.create({
+  smaller: { height: tokens["space-smaller"] },
```

Comment:
`small` 😉 change: no longer using [spacing for size values!](https://atlantis.getjobber.com/?path=/docs/guides-frontend-styleguide--docs#size-width-and-height)